### PR TITLE
Force single response selection in response correlation page

### LINKED ISSUE #254: You can select multiple responses but only one is plotted. It should restrict to only one selected. todo

### PR DESCRIPTION
**Pull request by DanSava**
**Issue**
Resolves #254

### CODE REVIEW

File: webviz_ert/plugins/_response_correlation.py [hunk 1]
```diff
@@ -93,8 +94,8 @@ def layout(self) -> Component:
                         dbc.Col(
                             [
                                 html.Label("Responses", className="ert-label"),
```
Comment (oysteoh) [inline webviz_ert/plugins/_response_correlation.py:96]: Maybe change to "Response" instead as it is one single selected now?

**Comment by oysteoh** [comment]
There is a "bug" which needs to be handled. It can be reproduced like this.

Navigate to `test-data/local/snake_oil/`
Launch `ert vis snake_oil.ert` 
Go to the response-correlation view and select `default_0`,  Response `BPR:445` and then one of the parameters.

response_correlation_viewer

```
ERROR:webviz_app:Exception on /_dash-update-component [POST]
Traceback (most recent call last):
  File "/datadisk/virtualenvs/3.8/lib64/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "/datadisk/virtualenvs/3.8/lib64/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/datadisk/virtualenvs/3.8/lib64/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "/datadisk/virtualenvs/3.8/lib64/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "/datadisk/virtualenvs/3.8/lib64/python3.8/site-packages/dash/dash.py", line 1372, in dispatch
    response.set_data(func(*args, outputs_list=outputs_list))
  File "/datadisk/virtualenvs/3.8/lib64/python3.8/site-packages/dash/_callback.py", line 151, in add_context
    output_value = func(*func_args, **func_kwargs)  # %% callback invoked %%
  File "/datadisk/virtualenvs/3.8/lib64/python3.8/site-packages/webviz_ert/controllers/response_correlation_controller.py", line 83, in update_correlation_plot
    parameter_df[selected_response] = response_df.iloc[x_index]
  File "/datadisk/virtualenvs/3.8/lib64/python3.8/site-packages/pandas/core/indexing.py", line 931, in __getitem__
    return self._getitem_axis(maybe_callable, axis=axis)
  File "/datadisk/virtualenvs/3.8/lib64/python3.8/site-packages/pandas/core/indexing.py", line 1566, in _getitem_axis
    self._validate_integer(key, axis)
  File "/datadisk/virtualenvs/3.8/lib64/python3.8/site-packages/pandas/core/indexing.py", line 1500, in _validate_integer
    raise IndexError("single positional indexer is out-of-bounds")
IndexError: single positional indexer is out-of-bounds

```

**Comment by DanSava** [comment]
> There is a "bug" which needs to be handled. It can be reproduced like this.
> 
> Navigate to `test-data/local/snake_oil/` Launch `ert vis snake_oil.ert` Go to the response-correlation view and select `default_0`, Response `BPR:445` and then one of the parameters.

Very strange, seems I was using an old instance of ert and I was not experiencing the bug you encountered, once I updated ert I was able to reproduce and fix it.

**Review by oysteoh — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**